src/main/resources/application.yml

```diff
@@ -2,7 +2,7 @@ server:
   port: 8081
 spring:
   datasource:
-    url: jdbc:postgresql://localhost:5432/db_wst
+    url: jdbc:postgresql://postgresqldb:5432/db_wst
```

Comment:
What this does?

Reply:
volumes -> it means that docker will be save postgres data localy even after delete images and conteiners
url: jdbc:postgresql://postgresqldb:5432/db_ws -> here I bind postgres image from docker